### PR TITLE
JDBC-UnitTest-ModularCode-Java

### PR DESCRIPTION
**Pull request by jubitjohn1**
Please go through the code and suggest

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ target/
 !**/src/test/**/target/
 
 ### IntelliJ IDEA ###
+.idea/
 .idea/modules.xml
```
Comment (adityapillai1) [inline .gitignore]: only .idea/ is needed

File: src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -7,39 +9,23 @@
...
 import java.util.List;
 import java.util.Properties;
+import java.util.logging.Logger;
+
 
 public class Main {
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
     public static void main(String[] args) throws SQLException, ClassNotFoundException {
 
-//        if (args.length != 1) {
```
Comment (adityapillai1) [inline src/main/java/org/example/Main.java:14]: should remove commented code.

File: src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -7,39 +9,23 @@
...
-//            System.err.println("Usage: java Main <file_path>");
-//            return;
-//        }
-
-        String configFilePath = "config.properties";
         String csvFilePath = "C:\\Users\\jubit.john\\IdeaProjects\\maven_project\\src\\main\\resources\\Demo.json";
```
Comment (adityapillai1) [inline src/main/java/org/example/Main.java]: replace full file path

Comment (jubitjohn1) [reply src/main/java/org/example/Main.java]: Okay

**Pull request merged**